src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,277 @@
+import java.util.Scanner;
+public class Duke {
+
+    public static void printBorder() {
+        String border = "____________________________________________________________ \n";
+        System.out.println(border);
+    }
+
+    public static void printGreeting(){
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        String border = "____________________________________________________________ \n";
+        String greeting = "Hello! I'm duke.Duke\n" + "What can I do for you?\n";
+        System.out.println(logo + border+ greeting);
+    }
+
+    //prints exit statement
+    public static void printExit(){
+        String exit = "Bye. Hope to see you again soon! \n";
+        System.out.println(exit);
+    }
+
+
+    //add a new to do
+    public static void addTodo(ArrayList<Task> tasks, String name) throws EmptyTaskException {
+        if(name.equals(" ")){
+            throw new EmptyTaskException();
+        }
+        tasks.add(new Todo(name, false));
+        printBorder();
+        System.out.println("added: " + name + "\n");
+        printBorder();
+    }
+
+    //add a new deadline task
+    public static void addDeadline(ArrayList<Task> tasks, String name, String deadline){
+        tasks.add(new Deadline(name, deadline, false));
+        printBorder();
+        System.out.println("added: " + name + "\n");
+        printBorder();
+    }
+
+    //add a new event
+    public static void addEvent(ArrayList<Task> tasks, String name, String start, String end){
+        tasks.add(new Event(name, start, end, false));
+        printBorder();
+        System.out.println("added: " + name + "\n");
+        printBorder();
+    }
+
+    //print list of duke.tasks
+    public static void printList(ArrayList<Task> tasks){
+        printBorder();
+
+        int counter = 1;
+        for (Task t : tasks) {
+            if (t == null) {
+                break;
+            } else {
+                System.out.println(counter + ". " + t);
+                counter++;
+            }
+        }
+        System.out.println("You have " + tasks.size() + " tasks in your list.");
+        printBorder();
+    }
+
+    //mark task as done
+    public static void markTask(ArrayList<Task> tasks, int taskIndex){
+        tasks.get(taskIndex).setStatus("mark");
+        System.out.println("Nice! I've marked this task as done: \n");
+        System.out.println(tasks.get(taskIndex) + "\n");
+        printBorder();
+    }
+
+    //mark task as undone
+    public static void unmarkTask(ArrayList<Task> tasks, int taskIndex){
+        tasks.get(taskIndex).setStatus("unmark");
+        System.out.println("OK, I've marked this task as not done yet: \n");
+        System.out.println(tasks.get(taskIndex) + "\n");
+        printBorder();
+    }
+
+//    delete task
```

Comment:
Perhaps making comments for those not so obvious code, do not need to comments on the obvious. You can refer to this: https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#do-not-repeat-the-obvious
For comment of the whole method you will learn the java doc annotation in week 7